content/development/coding-guidelines.md

```diff
@@ -79,9 +79,19 @@ Foo::Bar(const char* bar1, T bar2)
 	...
 }
 
-//  Empty functions converted to single line functions
+// Empty functions in class should be converted to inline functions
```

Comment:
"inline" means something specific in C and C++, to avoid confusion we should not use that word here.

```suggestion
// Empty functions in class definitions can be written on a single line
```